fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIOCatsConversions.scala

```diff
@@ -143,10 +147,31 @@ object BIOCatsConversions {
   }
 
   class BIOCatsSync[F[+_, +_]](override val F: BIO[F]) extends BIOCatsBracket[F](F) with cats.effect.Sync[F[Throwable, ?]] {
+    self =>
     @inline override final def suspend[A](thunk: => F[Throwable, A]): F[Throwable, A] = F.flatten(F.syncThrowable(thunk))
     @inline override final def delay[A](thunk: => A): F[Throwable, A] = F.syncThrowable(thunk)
   }
 
+  class BIOCatsParallel[F0[+_, +_]](override val F: BIO[F0], private val Par: BIOParallel[F0]) extends BIOCatsSync[F0](F) with cats.Parallel[F0[Throwable, ?]] {
```

Comment:
@neko-kai For `BIOCatsParallel` I had to add an extra param for `Par` since we need to override the `F` from `BIOCatsSync` and `BIOCatsSync` requires a `BIO` which `BIOParallel` does not provide. Is this the correct approach or is a better way of doing this?

Reply:
Oh, I misread it. I thought BIOParallel inherited from BIO, but it's actually disjoint. In that case you don't really need to inherit from any of the other classes, just
```scala
class BIOCatsParallel[F0[+_, +_]](private val F0: BIOParallel[F0]) extends cats.Parallel[F0[Throwable, ?]] {
```
Since cats.Parallel is disjoint from the hierarchy too, you may not even need a specificity trait.
You'll need the `InnerF` here to contain a BIOMonad instead of BIOFunctor, to implement `monad` - https://github.com/7mind/izumi/pull/1029/files#diff-9f1afb7987c8b71ef9edd0e0b8203de3R4
You'll also need to add an attachment for BIOParallel to F like this: https://github.com/7mind/izumi/blob/develop/fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIORoot.scala#L41
And a conversion from BIOParallel to BIOMonad, extracting InnerF, like this: https://github.com/7mind/izumi/blob/develop/fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIORoot.scala#L37